### DISCUSSION TITLE
Subscription not working. Not showing the extra features after the subscription.

### DISCUSSION THREAD

**Discussion by kalilmd** [emoacht/Monitorian#408] [Q&A] [answered]
I already did the Microsoft store subscription through the app. I closed the app and reopened it several times. I reloaded the subscription information several times. And I still don't get the extra add-on features.

How long does it take for my subscription to be activated and for me to have access to the add-on features?

Thanks

**Comment by kalilmd** [comment, marked as answer]
On the Windows 11 Settings > Apps > Installed apps > Monitorian > Advanced Options

I repaired then reset the app and restarted the PC and now it's working fine.

**Comment by emoacht** [comment]
Thanks for sharing information.
I have no idea what was the problem. If any file in installation package is missed or corrupted, most probably this app will not start at all.
I move this issue to the discussion for the reference for other users.

**Comment by emoacht** [comment]
FYI, if you find the subscription information is not updated, click `Reload subscription information`. It will make this app re-retrieve the subscription information from Microsoft Store server.

**Reply by Zlypkorp** [reply]
Hi @emoacht, this issue is still happening. I clicked several times on Reload subscription information but it doesn't work at all. I fexed the issue follwing kalilmd steps.

**Comment by emoacht** [comment]
Could you check the raw subscription information?

1. Update to Ver 4.7.4
2. Enable operation.log
3. Run this app for a couple of hours
4. Click "Reload subscription information"
5. Copy operation.log to the desktop
6. Find the part surrounded by "Store Responses Start" and "Store Responses End" and post it here.

How to produce operation.log is explained in [Reporting](https://github.com/emoacht/Monitorian#reporting) section of readme.
